src/pages/bosblog/index.tsx

```diff
@@ -26,7 +27,7 @@ const Blog: NextPageWithLayout = () => {
           requestAuthentication,
           accountId,
           blockHeight,
-          includeFooter: true,
+          blogComponentUrl,
```

Comment:
@jackson-harris-iii  Is the removal of `includeFooter` intentional? This was recently added

Reply:
Good catch - we should make sure we keep `includeFooter: true`.

Reply:
definitely accidental, corrected!